pkg/controller/performanceprofile/components/profile/profile_test.go

```diff
@@ -75,15 +77,15 @@ var _ = Describe("PerformanceProfile", func() {
 			Expect(err.Error()).To(ContainSubstring("hugepages default size should be equal"))
 		})
 
-		It("should reject hugepages allocation with different sizes", func() {
+		It("should reject hugepages allocation with unexpected page size", func() {
 			profile.Spec.HugePages.Pages = append(profile.Spec.HugePages.Pages, v1.HugePage{
 				Count: 128,
 				Node:  pointer.Int32Ptr(0),
-				Size:  v1.HugePageSize("2M"),
+				Size:  v1.HugePageSize("14M"),
```

Comment:
Can you explain this change?

Reply:
I changed the code to drop the validation once we have unsupported hugepages size and the same did for the unittest.

Reply:
You updated 2M to 14M somwhere?

Reply:
Isn't "14M" a random illegal value which should trigger the validation failure (which the tests expects and want to make sure it happens)?